SumerCamp2021/src/main/java/frc/robot/Robot.java

```diff
@@ -44,7 +44,7 @@ public static double voltageToDist(double voltage){
   @Override
   public void robotInit() {
     m_diffDrive.setSafetyEnabled(false);
-    m_diffDrive.setRightSideInverted(true);
+    m_diffDrive.setRightSideInverted(false);
```

Comment:
sets inverted as false, so motors will naturlly drive in opposite directions. This will be the first problem we have students solve